frappe/public/js/lib/leaflet/leaflet.css

```diff
@@ -11,7 +11,6 @@
 .leaflet-image-layer,
 .leaflet-layer {
 	position: absolute;
-	left: 0;
 	top: 0;
 	}
 .leaflet-container {

```

Comment:
@alaa-alsalehi Nice catch!

but you can simply add `/*rtl:begin:ignore*/` & `/*rtl:end:ignore*/` to the start and end of this block to avoid rtl conversion during build process.

check this for reference https://github.com/frappe/frappe/blob/develop/frappe/public/scss/common/quill.scss#L10

---
Once this is done other changes are not required.

Reply:
done check now @surajshetty3416